### PR TITLE
Pronoun tagging - beach any med

### PR DESCRIPTION
**Pull request by sedgestripe**

### CODE REVIEW

File: resources/dicts/patrols/beach/med/any.json [hunk 1]
```diff
@@ -1,758 +1,1153 @@
...
+    {
+        "patrol_id": "bch_med_soloripples1",
+        "biome": "beach",
+        "season": "Any",
+        "tags": [
+            "clan_to_p_l",
+            "clan_to_p_l",
+            "clan_to_p_l",
+            "clan_to_p_l",
+            "clan_to_p_l",
```
Comment (keyraven) [inline resources/dicts/patrols/beach/med/any.json]: The pronoun-tagging script seems to have messed with the tags on this patrol.

File: resources/dicts/patrols/beach/med/any.json [hunk 2]
```diff
@@ -1,758 +1,1153 @@
...
+    },
+    {
+        "patrol_id": "bch_med_mossapps8",
+        "biome": "beach",
+        "season": "Any",
+        "tags": [
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
```
Comment (keyraven) [inline resources/dicts/patrols/beach/med/any.json]: This patrol also has mangled tags.

File: resources/dicts/patrols/beach/med/any.json [hunk 3]
```diff
@@ -1,758 +1,1153 @@
...
+    },
+    {
+        "patrol_id": "bch_med_mossapps6",
+        "biome": "beach",
+        "season": "Any",
+        "tags": [
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
```
Comment (keyraven) [inline resources/dicts/patrols/beach/med/any.json]: Messed up tags here as well.

File: resources/dicts/patrols/beach/med/any.json [hunk 4]
```diff
@@ -1,758 +1,1153 @@
...
+        "patrol_id": "bch_med_mossapps1",
+        "biome": "beach",
+        "season": "Any",
+        "tags": [
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
+            "p_l_to_r_c",
```
Comment (keyraven) [inline resources/dicts/patrols/beach/med/any.json]: It seems any patrol with "p_l_to_r_c" as a tag got mangled

**Review by keyraven — CHANGES_REQUESTED** [review]
This looks pretty good! Just some weird tag stuff with at least one patrol.

**Comment by sedgestripe** [comment]
> This looks pretty good! Just some weird tag stuff with at least one patrol.

Thanks! Will have to look into why it is duplicating some tags

**Comment by keyraven** [comment]
The tags weren't just duplicated: Some of the existing tags were also deleted. For example, it remove the "herb-gathering" tag from most of the patrols where the tag was duplicated.

**Comment by sedgestripe** [comment]
Whoops, will fix that now

**Pull request merged**